### ISSUE TITLE
[request] Support direct boot without flash

### ISSUE THREAD

**Issue by Azkali** [Benjamin-Dobell/Heimdall#515]
Hi,
I would like to know if an equivalent to `fastboot boot` command would be feasible in heimdall ?

**Comment by Grimler91** [comment]
No, once in download mode the bootloader side, "loki", does not have such functionality so such a thing is not possible through heimdall.

You might be interested in reading about "usb download mode" ([here](https://github.com/astarasikov/exynos9610-usb-emergency-recovery) and [here](https://github.com/frederic/exynos-usbdl)) that at least some models seem to support, and about the available commands in the [sboot shell (through uart connection)](https://hexdetective.blogspot.com/2017/02/exploiting-android-s-boot-getting.html), and about how people managed to put [sboot on sdcard and boot it in the galaxy s3](https://www.xda-developers.com/fix-your-soft-bricked-galaxy-s-iii-samsungs-way/)
